test/test_quantized_tensor.py

```diff
@@ -193,6 +193,38 @@ def test_qtensor_copy(self):
         self.assertEqual(q.q_scale(), q2.q_scale())
         self.assertEqual(q.q_zero_point(), q2.q_zero_point())
 
+    def test_qtensor_clone(self):
```

Comment:
I don't think we need to test these cases, clone returns a new Tensor right?

I tested `copy_` for different scale and zero_point because it is inplace

Reply:
Yeah, makes sense.